images/build/debian-base/Makefile

```diff
@@ -92,10 +92,15 @@ else
 endif
 	mv $(TEMP_DIR)/$(CONFIG)/Dockerfile.build.tmp $(TEMP_DIR)/$(CONFIG)/Dockerfile.build
 
-	docker build --pull -t $(BUILD_IMAGE) -f $(TEMP_DIR)/$(CONFIG)/Dockerfile.build $(TEMP_DIR)/$(CONFIG)
+	docker buildx build \
```

Comment:
We may run into issues here with pulling the base image when using buildkit and gcr mirror (see https://github.com/kubernetes/kubernetes/pull/97982 for more info)